app/controllers/accounts_controller.rb

```diff
@@ -16,9 +16,11 @@ def show
       sp_session_request_url: sp_session_request_url_with_updated_params,
       authn_context: resolved_authn_context_result,
       sp_name: decorated_sp_session.sp_name,
+      sp_handoff_already_occurred: session[:sp_successful_handoff],
       user: current_user,
       locked_for_session: pii_locked_for_session?(current_user),
     )
+    session.delete(:sp_successful_handoff)
```

Comment:
If the user refreshed the page, would the banner start showing again since we delete the session value?

Reply:
hmm good point. I think we can keep it until sign off? And it may be better to store it in the sp_session then.